current-events/activity-current-events.md

```diff
@@ -0,0 +1,160 @@
+
+Everyone will prepare a presentation of three or less slides to introduce a current event topic of your choice.
+
+## Preparation
+
+A slide deck of three or fewer slides should be prepared before activity day. Slide deck requirements are below.
+
+On the day of presentations, we should be prepared to discuss, present, and share in small groups.
+
+Open discussion about current events can affect us in many ways. Everyone should be prepared to engage in discussions however is best for you. This includes disengaging!
+
+## Activity Instructions
+
+1. Find an article/piece of media that describes a current event about technology that interests you.
+
+Current events should be topics actively relevant within the last twelve months. Some examples include:
+
+- A recent news article describing how tech is transforming
+- A documentary on some technology that was built in the last year
+- A thinkpiece on technology written recently
+- Instance of action, resistance, and/or justice related to technology and current events
+
+Avoid fiction, unless it is unambiguously making a direct connection to a current event.
+
+2. Create a slide deck of three or less slides describing the current event.
+
+3. On activity day, follow the facilitator's directions to get into small discussion groups.
+
+4. Within the discussion groups, take turns so that every person introduces their current event topic with their own slide deck.
+
+5. After everyone has presented their current event topic, pick a current event to begin the discussion. Then, work together as a team to discuss the following:
+
+    - How does this news make you feel? Do you feel:
+        - celebratory?
+        - encouraged?
+        - angry?
+        - happy or pleased?
+        - sad?
+        - curious?
+        - amazed?
+        - hungry?
+    - Who are the people most involved or affected by this?
+    - Does this affect me? Why or why not?
+    - How do we learn more?
+
+6. When the conversation has come to a stop, move to the next topic.
+
+7. Congratulate you and your team for engaging in critical discussion! This requires openness, vulnerability, and effort.
+
+## Slide Deck Requirements
+
+- Your name and pronouns
+- A link to the article/piece of media
+- A description of the contents of the article/piece of media
+- A graphic/photograph/picture/logo
+- Answer this question: Why did you think it was interesting?
+- A content warning slide listing all content warnings if relevant (details below)
+
+### !callout-info
+
+## Separation From Current Events
+
+The current news we bring and discuss is not necessarily a reflection of who we are.
+
+### !end-callout
+
+### !callout-success
+
+## Keep Each Other Safe. Keep Yourself Safe.
+
+The safety of marginalized identities in tech is the priority of this activity.
+
+### !end-callout
+
+### !callout-info
+
+## Getting Started
+
+If you're unsure where to begin research, we can begin with finding recent news any of these topics:
```

Comment:
instructions, so voiced to you
removed redundant sounding second begin phrase
missing about
```suggestion
If you're unsure where to begin your research, try finding recent news about any of these topics:
```